Subject: [PATCH] ANR related to split-screen

In some cases, for example, when an activity with a dialog leaves
split-screen mode, the input focus is not changed to the recreated
window, tap Back key, anr is coming.
To avoid this, update window focus one more time.

diff --git a/services/core/java/com/android/server/wm/WindowManagerService.java b/services/core/java/com/android/server/wm/WindowManagerService.java
index 3687512..5e7f8fe 100644
--- a/services/core/java/com/android/server/wm/WindowManagerService.java
+++ b/services/core/java/com/android/server/wm/WindowManagerService.java
@@ -2462,6 +2462,7 @@
                 // animation.
                 win.mAnimatingExit = true;
                 win.mReplacingRemoveRequested = true;
+                updateFocusedWindowLocked(UPDATE_FOCUS_WILL_PLACE_SURFACES, true);
                 Binder.restoreCallingIdentity(origId);
                 return;
             }
